src/app/home/form/form.component.css

```diff
@@ -0,0 +1,518 @@
+/* /* .form-box{
+  width:50vh;
+  padding: 15px;
+  border: none;
+  border-radius: 5px;
+   box-shadow:
+  0 2.8px 2.2px rgba(0, 0, 0, 0.034),
+  0 6.7px 5.3px rgba(0, 0, 0, 0.048),
+  0 12.5px 10px rgba(0, 0, 0, 0.06),
+  0 22.3px 17.9px rgba(0, 0, 0, 0.072),
+  0 41.8px 33.4px rgba(0, 0, 0, 0.086),
```

Comment:
This file could be shortened up,removing unused CSS can help improve SEO

Reply:
will do it